src/nyc_trees/apps/core/tasks.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+
+import os
+import uuid
+import importlib
+import tempfile
+
+from django.core.files.storage import default_storage
+
+from djqscsv import write_csv
+
+# Using strings so we can easily transition to async tasks, which need
+# serializable arguments
+MODELS = ['apps.core.models.User',
+          'apps.core.models.Group',
+          'apps.event.models.Event',
+          'apps.event.models.EventRegistration',
+          'apps.survey.models.BlockfaceReservation',
+          'apps.survey.models.Species',
+          'apps.survey.models.Survey',
+          'apps.survey.models.Territory',
+          'apps.users.models.Achievement',
+          'apps.users.models.Follow',
+          'apps.users.models.TrainingResult',
+          'apps.users.models.TrustedMapper']
+
+
+def dump_model(fq_name, dump_id):
+    Model = _model_from_fq_name(fq_name)
+    _, temp_file_path = tempfile.mkstemp()
```

Comment:
We should make sure to delete this temp file after storing it in the media directory / S3, to avoid filling up the disk.